docs/core/entity/fan.md

```diff
@@ -16,17 +16,31 @@ Properties should always only return information from memory and not do I/O (lik
 | current_direction | str | None | Return the current direction of the fan |
 | is_on | boolean | None |Return true if the entity is on |
 | oscillating | boolean | None | Return true if the fan is oscillating |
+| percentage | int | None | Return the current speed percentage. Must be a value between 0 (off) and 100 |
+| supported_features | int | 0 | Flag supported features |
+| preset_mode | str | None | Return the current preset_mode. One of the values in preset_modes. |
+| preset_modes | list | None | Get the list of available preset_modes. This is an arbitrary list of str and should not contain any speeds. |
+
+## Deprecated Properties
+
+The fan entity model has changed to use percentages in the range from 0 (off) to 100 instead
+of the named speeds. The new model replaces `speed` and `speed_list` with `percentage`, `preset_mode`, and `preset_modes`. This change allowed us to expand the number of supported speeds to accommodate additional fan models in Home Assistant. 
+
+To maintain backwards compatibility with integations that have not updated to the new model, the deprecated properties will remain until at least the end of 2021. Integrations must update their the [Turn on](#turn-on) function to consume `percentage` or `preset_mode` instead of `speed`.
```

Comment:
```suggestion
To maintain backwards compatibility with integations that have not updated to the new model, the deprecated properties will remain until at least the end of 2021. Integrations must update their [Turn on](#turn-on) function to consume `percentage` or `preset_mode` instead of `speed`.
```